operator/templates/cassandra-env-sh.yaml

```diff
@@ -251,8 +251,9 @@ data:
 
     {{ if ne $.Params.JMX_LOCAL_ONLY "true" }}
       JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.local.only=false"
-      JVM_OPTS="$JVM_OPTS -Djava.rmi.server.hostname=$POD_NAME.{{ .Name }}-svc.$POD_NAMESPACE.svc.cluster.local"
-      
+      #JVM_OPTS="$JVM_OPTS -Djava.rmi.server.hostname=$POD_NAME.{{ .Name }}-svc.$POD_NAMESPACE.svc.cluster.local"
+      JVM_OPTS="$JVM_OPTS -Djava.rmi.server.hostname=$POD_IP"
+
```

Comment:
This might get undone later, it's to try another version to shutdown the old cassandra instance